types/google-map-react/utils.d.ts

```diff
@@ -0,0 +1,22 @@
+import { Bounds, Coords, Point, Size } from '.';
+
+interface tile extends Point {
```

Comment:
I recommend calling this `Tile` - types are usually capitalized.

Reply:
Agreed and updated